Dockerfile

```diff
@@ -121,6 +121,8 @@ RUN --mount=type=bind,source=./docker,target=/docker \
         libecpg-dev \
         libldap2-dev \
     && touch superset/static/version_info.json \
+    && chown -R superset:superset /app \
+    && chmod -R 775 /app \
```

Comment:
wondering if these 2 commands would be belong right before the `USER superset` command, and if we do so we might be able to simplify the `COPY` commands and other chown/chmod throughout the Dockerfile

Reply:
https://github.com/apache/superset/pull/31385